pages/linux/ntpd.md

```diff
@@ -0,0 +1,8 @@
+# ntpd
+
+> The official Network Time Protocol daemon program.
+> More information: <https://www.ntp.org/documentation/4.2.8-series/ntpd/>.
+
```

Comment:
```suggestion

- Start the daemon:

`sudo ntpd`
```
We prefer to start with a basic usage and progressively increase the complexity.